src/index.ts

```diff
@@ -55,6 +55,9 @@ const main = async () => {
   }
   const { oauthClient, oauthClientSecret, dtEnvironment, slackConnectionId } = dynatraceEnv;
 
+  // create an oauth-client
+  const dtClient = await createOAuthClient(oauthClient, oauthClientSecret, dtEnvironment, scopesBase);
```

Comment:
We moved `dtClient` into tools with #45  - You can add an initial call for setup, but we need to differentiate which `dtClient` is used.